torch/onnx/symbolic_opset9.py

```diff
@@ -1159,11 +1173,19 @@ def log_softmax(g, input, dim, dtype=None):
 @parse_args('v', 'v', 'v', 'is', 'is', 'is', 'i', 'is', 'i', 'i', 'i', 'i', 'i')
 def _convolution(g, input, weight, bias, stride, padding, dilation,
                  transposed, output_padding, groups, benchmark, deterministic, cudnn_enabled, allow_tf32):
-    weight_size = weight.type().sizes()
+    weight_size = sym_help._get_tensor_sizes(weight)
+    try:
```

Comment:
Why try/catch here? Can we check if weight_size is not None and rank >= 2?

Reply:
try-except is less verbose, more pythonic in this simple case.